src/features/Apiexplorer/LoginDialog/index.tsx

```diff
@@ -57,7 +62,7 @@ export const LoginDialog = ({ setToggleModal }: TLoginDialog) => {
               <Button color='tertiary' onClick={handleSignUp} className={styles.btn}>
                 <Translate>Sign up</Translate>
               </Button>
-              <Button color='primary' onClick={handleClick} className={styles.btn}>
+              <Button color='primary' onClick={handleLogin} className={styles.btn}>
```

Comment:
[Q] do we not have feature flag to toggle between phase 1 and 2?

Reply:
Handled in the useHandleLogin hook 🙏